_security-plugin/access-control/multi-tenancy.md

```diff
@@ -56,7 +56,7 @@ opensearch_security.multitenancy.enable_filter: false
 
 Setting | Description
 :--- | :---
-`opensearch.requestHeadersWhitelist` | OpenSearch Dashboards requires that you whitelist all HTTP headers that it passes to OpenSearch. Multi-tenancy uses a specific header, `securitytenant`, that must be present with the standard `Authorization` header. If the `securitytenant` header is not whitelisted, OpenSearch Dashboards starts with a red status.
+`opensearch.requestHeadersallowlist` | OpenSearch Dashboards requires that you allowlist all HTTP headers that it passes to OpenSearch. Multi-tenancy uses a specific header, `securitytenant`, that must be present with the standard `Authorization` header. If the `securitytenant` header is not allowlisted, OpenSearch Dashboards starts with a red status.
```

Comment:
According to the style guide, I think we want to avoid using allow list as a verb, and that it should be two words.

Reply:
I noticed [AWS Style Guide](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/dictionary.html#style-A-terms) separates it for noun usage, but wiktionary still shows "allowlist" as a [verb](https://en.wiktionary.org/wiki/allowlist#:~:text=Synonym%20of%20whitelist-,Verb,past%20and%20past%20participle%20allowlisted)).

Reply:
Switched to "If the `securitytenant` header is not on the allowlist, OpenSearch Dashboards starts with a red status."

Reply:
Switched to "If the `securitytenant` header is not on the allowlist, OpenSearch Dashboards starts with a red status."